server/utils/FirebaseAlgoliaServer/functions/tests/integrationTestHelpers.js

```diff
@@ -0,0 +1,311 @@
+// Initialize constants, libraries
```

Comment:
Please organize this file better.  In general it is hard to read because there are so many methods doing similar things with similar names.

Group related functions

Create abstract functions that can flexibly handle multiple different use cases, instead of repeated new methods that basically do the same thing in individual use cases

Try to reduce the number of methods via parameters or passing around functions